main.cpp

```diff
@@ -193,7 +193,7 @@ struct SwissArmyKnife
 
 void SwissArmyKnife::cut(Knife::Incision aNewCut)
 {
-    aNewCut.hurt(1);
+    aNewCut.getHurtLevel();
```

Comment:
If you're going to call this function, since it returns a value, why not print out that value via std::cout?  
- [ ] revise